### PR TITLE
Update Feature List and Parameter Details

### LINKED ISSUE #179: Update Feature List in UG to accommodate Medical Condition Management feature
Currently the feature list does not include the our new Medical Condition Management feature

- Add Medical Condition
- Delete Medical Condition

We will add these into the UG's feature list in this issue.

### LINKED ISSUE #189: Update Parameter Details to include CONDITION
Currently Parameter Details does not include `CONDITION`, which is a parameter for our commands under Medical Condition Management. Will add it in this issue.

### LINKED ISSUE #190: Update Parameter Details to include PRIORITY
Currently Parameter Details does not include PRIORITY, which is a parameter for our commands under Priority Management. Will add it in this issue.

### PR DESCRIPTION
**Pull request by PresleyChew**
Updated Feature List to include Medical Condition Management features with the following commands

- `addMedCon`
- `delMedCon`

Updated Parameter Details to include the following Parameters

- `PRIORITY`
- `CONDITION`

fixes #189 , fixes #190 , fixes #179

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -232,7 +234,50 @@ to `91234567` and `johndoe@example.com` respectively.
...
+* `addMedCon i/S1234567A c/High Blood Pressure` will add medical condition `High Blood Pressure` to patient with NRIC `S1234567A`.   
+
+{: .alert .alert-success}
+> :bulb: **Tip:**
+>
+> User can add more than 1 Medical Condition through using `c/CONDITION` multiple times
+
+#### Deleting Medical Conditions : `delMedCon`
+
+Deletes Medical Condition from an existing patient in MediBase3.
```
Comment (yong-zhuo) [inline docs/UserGuide.md:263]: I think can keep 'medical condition' in lowercase to remain consistent with the addMedCon command

File: docs/UserGuide.md [hunk 2]
```diff
@@ -232,7 +234,50 @@ to `91234567` and `johndoe@example.com` respectively.
 ### Managing Appointments
 [To be filled up]
 ### Managing Medical Conditions
-[To be filled up]
+
+#### Adding Medical Conditions : `addMedCon`
+
+Adds medical condition to an existing patient in MediBase3.
+
+Format: `addMedCon i/NRIC c/CONDITION`
```
[the remark this answers is not in the document]
Comment (PresleyChew) [reply docs/UserGuide.md]: Amended

File: docs/UserGuide.md (hunk 1 above)
Comment (PresleyChew) [reply docs/UserGuide.md:263]: Good catch! Will make "medical condition" in addMedCon command to be upper caps as well to ensure consistency, viewing it as an object, hence the upper caps.

File: docs/UserGuide.md [hunk 3]
```diff
@@ -232,7 +234,54 @@ to `91234567` and `johndoe@example.com` respectively.
...
+> * You can add a medical condition to a patient even if they are not being currently displayed in the Patient List Panel but doing so will refresh the panel to display all patients after medical condition has been added.
+> * Refer to the [Parameter Details](#parameter-details) section for more information on the purpose and constraints of each parameter.
+
+Example: 
+* `addMedCon i/S1234567A c/High Blood Pressure` will add medical condition `High Blood Pressure` to patient with NRIC `S1234567A`.   
+
+{: .alert .alert-success}
+> :bulb: **Tip:**
+>
+> User can add more than 1 Medical Condition through using `c/CONDITION` multiple times
```
Comment (yadobler) [inline docs/UserGuide.md:257]: I think can add an example here
```md
{: .alert .alert-success}
> :bulb: **Tip:**
>
> User can add more than 1 Medical Condition through using `c/CONDITION` multiple times:
> `addMedCon i/S1234567C c/High Blood Pressure c/Osteoporosis`
```

**Review by yadobler — COMMENTED** [review]
LGTM

**Review by yong-zhuo — APPROVED** [review]

File: docs/UserGuide.md (hunk 3 above)
Comment (PresleyChew) [reply docs/UserGuide.md]: Added for both add and del!

**Review by yadobler — APPROVED** [review]
👍🏾

**Pull request merged**